core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java

```diff
@@ -2862,6 +2862,12 @@ public WriterScalingOptions getInsertWriterScalingOptions(Session session, Table
         return metadata.getInsertWriterScalingOptions(session.toConnectorSession(tableHandle.catalogHandle()), tableHandle.connectorHandle());
     }
 
+    @Override
+    public void setEntityAuthorization(Session session, String ownedKind, List<String> name, TrinoPrincipal principal)
```

Comment:
Remove all other `set*Authorization` methods from this interface and implementations.

Reply:
Removed.